db/migrate/20171123105000_remove_last_checked_at_from_monitor_links.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveLastCheckedAtFromMonitorLinks < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :monitor_links, :last_checked_at, :datetime
```

Comment:
You'll also need to remove the field from https://github.com/alphagov/link-checker-api/blob/master/spec/factories/monitor_links.rb#L3 - might be worth adding a test so this would have been caught.

Reply:
What do you think the value of the test would be? Just because it seems like it wouldn't be testing the behaviour, rather the absence of a field?

Reply:
I was just surprised because it means there is no test which creates a `MonitorLink` factory, so if another developer tries to use that factory it will fail and could be confusing as to why.

A pattern we've used in a number of apps is to create a test that just ensures the default factory is valid: https://github.com/alphagov/email-alert-api/blob/master/spec/models/subscriber_spec.rb#L7